caprover/stack_deploy.py

```diff
@@ -305,13 +298,11 @@ def deploy_stack(config, gc_repository, dry_run):
 
             cap.update_app(
                 app_name,
-                # set persistent volume mount to /mnt/persistent-storage
                 persistent_directories=[
-                    "files-database:/database",
-                    "/mnt/persistent-storage:/persistent-storage",
+                    f"{app_name}-database:/database",
+                    f"{app_name}-config:/config",
+                    "/mnt/persistent-storage:/srv",  # The files to be served up live here
```

Comment:
What behavior have we been using on legacy deploys? Should Filebrowser expose the root of the persistent storage share, or a subfolder called `datalake/`?  @rudokemper

Reply:
A subfolder called `datalake/`. As you know we sometimes (awkwardly) use the root of the persistent storage share to also handle tmp files.

Reply:
I found a workaround: we can pass the root (-r) via an env variable.

The result:
- filebrowser will start
- when you  login you will see the following:
    <img width="703" height="543" alt="image" src="https://github.com/user-attachments/assets/843d5162-cb5b-4c07-b76b-5d42a63626b8" />
- Once you create the datalake folder using azure tools, you will correctly see an empty folder
- OR you can use the "New folder" button in the filebrowser webapp. Say you create a folder "abc" then it will implicitly create `datalake/abc` for you.
- In either case, the name "datalake" is NEVER exposed to the filebrowser user.

Reply:
Honestly I'm no longer confident whether this  used to work any better with the old Docker image. i.e. did "datalake" always have to exist before too?

Reply:
This seems fine. Could we make a note of the current behavior, and the possible workarounds (create using azure tools, or add a new folder in Filebrowser) in https://github.com/ConservationMetrics/gc-deploy/blob/main/caprover/INSTALL_GC_STACK.md#after-install?

Reply:
I see you've added a comment about it in lines 306-307, but it'd be good to expose in the README, as a sysadmin might be working through that document during setup and not think to check the script.